modules/wc-product-editor/module.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Elementor\Modules\WcProductEditor;
+
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\Plugin;
+use Elementor\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Module extends BaseModule {
+
+	public function __construct() {
+		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_assets' ] );
+		add_action( 'admin_footer', [ $this, 'print_button_js_template' ] );
+	}
+
+	public static function is_active() {
+		return self::is_new_woocommerce_product_editor_page();
+	}
+
+	public function enqueue_assets() {
+		$suffix = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
+
+		wp_enqueue_script( 'elementor-wc-product-editor', ELEMENTOR_ASSETS_URL . 'js/wc-product-editor' . $suffix . '.js', [], ELEMENTOR_VERSION, true );
+		wp_enqueue_style(
+			'elementor-wc-product-editor',
+			ELEMENTOR_ASSETS_URL . 'css/wc-product-editor' . $suffix . '.css',
+			[],
+			ELEMENTOR_VERSION
+		);
+	}
+
+	public function get_name() {
+		return 'wc-product-editor';
+	}
+
+	public static function is_new_woocommerce_product_editor_page() {
```

Comment:
Can you make sure this logic inside this method matches the logic in WooCommerce core? At least the parts we need.

```
protected function is_product_editor_page( $url ) {
		$query_string = wp_parse_url( wp_get_referer(), PHP_URL_QUERY );
		parse_str( $query_string, $query );

		if ( ! isset( $query['page'] ) || 'wc-admin' !== $query['page'] || ! isset( $query['path'] ) ) {
			return false;
		}

		$path_pieces = explode( '/', $query['path'] );
		$route       = $path_pieces[1];

		return 'add-product' === $route || 'product' === $route;
}
```